Examples/Heisenberg1d/heisenberg1d.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netket as nk
+import numpy as np
+import jax
+
+# 1D Lattice
+L = 20
+
+g = nk.graph.Chain(length=L, pbc=True)
+# Lattice translation operations
+
+# Hilbert space of spins on the graph
+hi = nk.hilbert.Spin(s=1 / 2, N=g.n_nodes, total_sz=0)
+
+# Ising spin hamiltonian
+ha = nk.operator.Heisenberg(hilbert=hi, graph=g)
+
+# RBM Spin Machine
+ma = nk.models.RBMSymm(
+    permutations=g.periodic_translations(),
+    alpha=4,
+    use_visible_bias=False,
+    use_bias=True,
```

Comment:
maybe we can say use_hidden_bias to avoid confusion with the old API?

Reply:
or ideally use_bias activates both... but OK...

Reply:
We can use `use_hidden_bias` - in that case we should update it on the `RBM` as well.

Reply:
This PR should now update `use_bias` to `use_hidden_bias` on all RBMs.

Reply:
The reason for using use_bias was that dense layers use that, so I was trying to be consistent with the rest of the ecosystem.

Reply:
yes true... but then use_visible_bias becomes problematic... :) I think it's fine to use RBM specific nomenclature